test/verify/check-machines

```diff
@@ -206,12 +225,33 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         return args
 
     def testBasic(self):
+        self._testBasic()
+
+    def testBasicLibvirtUserUnprivileged(self):
+        m = self.machine
+        user_group = 'libvirt'
+        user = createUser(m, user_group)
+        self.addCleanup(m.execute, "userdel -fr {0}".format(user))
```

Comment:
No need for that, nondestructive tests already clean up NSS and /home.

Reply:
Done